trikotFoundation/src/iosArm32Main/kotlin/com/mirego/trikot/foundation/strings/NativeStringExtensions.kt

```diff
@@ -0,0 +1,10 @@
+package com.mirego.trikot.foundation.strings
+
+import platform.Foundation.NSDiacriticInsensitiveSearch
+import platform.Foundation.NSString
+import platform.Foundation.create
+import platform.Foundation.stringByFoldingWithOptions
+
+actual fun String.normalize(): String =
+    NSString.create(string = this)
```

Comment:
`NSString.create(string = this)` est necessaire?

Reply:
Je n'ai pas trouvé d'autre moyen. `.stringByFoldingWithOptions` s'attend à être excecuté sur un NSString absolument, et il ne semble pas avoir d'autre façon de transformer une string kotlin en NSString autre qu'en utilsant `NSString.create(string = this)`.